bindings/java/org/collectd/java/GenericJMXConfMBean.java

```diff
@@ -242,6 +280,22 @@ public int query (MBeanServerConnection conn, PluginData pd, /* {{{ */
 
     return (0);
   } /* }}} void query */
+
+  private String join (String separator, List<String> list) /* {{{ */
```

Comment:
This just duplicates the join method in the GenericJMXConfValue class.  Just make it a common util function in a singleton class instead so you don't have the duplication.